src/main/io/serial.h

```diff
@@ -91,7 +91,8 @@ typedef enum {
     SERIAL_PORT_USB_VCP = 20,
     SERIAL_PORT_SOFTSERIAL1 = 30,
     SERIAL_PORT_SOFTSERIAL2,
-    SERIAL_PORT_IDENTIFIER_MAX = SERIAL_PORT_SOFTSERIAL2
+    SERIAL_PORT_IDENTIFIER_MAX = SERIAL_PORT_SOFTSERIAL2,
```

Comment:
`SERIAL_PORT_IDENTIFIER_MAX` is now ambiguos, as it is not actually the maximum possible value any more. Maybe better as `SERIAL_PORT_LAST` to avoid future confusion?